<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>U2::SaveSelectedSequenceFromMSADialog</class>
 <widget class="QDialog" name="U2::SaveSelectedSequenceFromMSADialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>179</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export Selected Sequence(s)</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="folderLabel">
       <property name="text">
        <string>Export to folder</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="folderNameEdit"/>
       </item>
       <item>
        <widget class="QToolButton" name="folderButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="fileNameLabel">
       <property name="text">
        <string>File name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="fileNameCombo"/>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="customFileNameEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="formatLabel">
       <property name="text">
        <string>File format</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="formatCombo"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="keepGapsBox">
     <property name="text">
      <string>Keep gaps</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="addToProjectBox">
     <property name="text">
      <string>Add to project</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>395</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>U2::SaveSelectedSequenceFromMSADialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>U2::SaveSelectedSequenceFromMSADialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
